addons/web_editor/static/lib/odoo-editor/src/qweb/Qweb.js

```diff
@@ -0,0 +1,212 @@
+/** @odoo-module **/
+import { ancestors } from '../utils/utils.js';
+
+export class QWebPlugin {
+    constructor(editor) {
+        this._editor = editor;
+        this._tGroupCount = 0;
+        this._hideBranchingSelection = this._hideBranchingSelection.bind(this);
```

Comment:
Ca manque un peu de commentaires/doc pour ces variables sur `this` quand même... ;-)